linter/allowlists/default.txt

```diff
@@ -467,3 +467,33 @@ Zipuploads
 Mimir
 Exception
 Workspace
+WorldOfImages
+Unsplash
+Bilddatenbanken
```

Comment:
All the German keywords could be added to the German allowlist instead: https://github.com/contao/package-metadata/blob/main/linter/allowlists/de.txt